screens/FavPokemon.tsx

```diff
@@ -1,52 +1,28 @@
-import React, { useEffect, useState } from 'react';
-import { StyleSheet, Text, View, Image, Button } from 'react-native';
-import getFavPokemon from '../api/pokemon';
-import styles from '../style/base';
-import PropRow from '../components/PropRow';
+import React from 'react';
```

Comment:
I'd keep stacks and screens separate.

If you don't want to keep all the logic in `App.tsx` (and that's nice, one big messy file is usually not the best idea) then I'd create a separate directory -> `containers`/`navigation`/`stacks` -> up to you, and put it there.